pkg/netflow/flowaggregator/flowstore.go

```diff
@@ -0,0 +1,56 @@
+package flowaggregator
+
+import (
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"github.com/DataDog/datadog-agent/pkg/netflow/common"
+)
+
+type flowStore struct {
+	flows map[string]*common.Flow
+	mu    sync.Mutex
+}
+
+func newFlowStore() *flowStore {
+	return &flowStore{
+		flows: make(map[string]*common.Flow),
+	}
+}
+
+func (f *flowStore) getFlows() []*common.Flow {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏This function also purges the accumulated flows... Maybe `flush()` would be a better name...

Reply:
+1, fixed here https://github.com/DataDog/datadog-agent/pull/10976/commits/21d0e7d8061c33a5f32d6a31b36461e4af99eafc